deployments/helm/KubeArmor/templates/deployment.yaml

```diff
@@ -22,6 +22,11 @@ spec:
       - image: {{printf "%s:%s" .Values.kubearmorRelay.image.repository .Values.kubearmorRelay.image.tag}}
         imagePullPolicy: {{ .Values.kubearmorRelay.imagePullPolicy }}
         name: kubearmor-relay-server
+        env:          
```

Comment:
These env vars in the generated manifest will look like:
![image](https://github.com/kubearmor/KubeArmor/assets/54525605/b64ef30b-7bcc-4532-9490-666aebe8436a)

Reply:
For similar example, please take a look at how we are embedding `args` from values.yaml into KubeArmor's daemonset.yaml template.